docs/config_search.md

```diff
@@ -14,88 +14,160 @@ See the License for the specific language governing permissions and
+---
 
-### Automatic Brute Search
+## Automatic Brute Search
 
-Automatic configuration search is the default behavior when running Model
-Analyzer without manually specifying what values to search. The parameters
-that are automatically searched are
-[`max_batch_size`](https://github.com/triton-inference-server/server/blob/master/docs/user_guide/model_configuration.md#maximum-batch-size)
-and
-[`instance_group`](https://github.com/triton-inference-server/server/blob/master/docs/user_guide/model_configuration.md#instance-groups).
-Additionally, [`dynamic_batching`](https://github.com/triton-inference-server/server/blob/master/docs/user_guide/model_configuration.md#dynamic-batcher) will be enabled if it is legal to do so.
+**Default brute search mode when no model config parameters are specified**
 
-An example model analyzer config that performs automatic config search looks
-like below:
+The parameters that are automatically searched are
+[model maximum batch size](https://github.com/triton-inference-server/server/blob/master/docs/user_guide/model_configuration.md#maximum-batch-size),
+[model instance groups](https://github.com/triton-inference-server/server/blob/master/docs/user_guide/model_configuration.md#instance-groups), and [request concurrencies](https://github.com/triton-inference-server/server/blob/master/docs/user_guide/perf_analyzer.md#request-concurrency).
+Additionally, [dynamic_batching](https://github.com/triton-inference-server/server/blob/master/docs/user_guide/model_configuration.md#dynamic-batcher) will be enabled if it is legal to do so.
+
+_An example model analyzer YAML config that performs an Automatic Brute Search:_
 
 ```yaml
 model_repository: /path/to/model/repository/
 
 profile_models:
-  - model_1
-  - model_2
+  - model_A
 ```
 
-In the default mode, automatic config search will try values 1 through 5 for
-[`instance_group`](https://github.com/triton-inference-server/server/blob/master/docs/user_guide/model_configuration.md#instance-groups).
-The maximum value can be changed using the `run_config_search_max_instance_count` key in the Model Analyzer Config.
-For each `instance_group`, Model Analyzer will sweep values 1 through 128 increasing exponentially (i.e. 1, 2, 4, 8, ...) for
-[`max_batch_size`](https://github.com/triton-inference-server/server/blob/master/docs/user_guide/model_configuration.md#maximum-batch-size). The start and end values can be changed using `run_config_search_min_model_batch_size` and `run_config_search_max_model_batch_size`.
-[`Dynamic_batching`](https://github.com/triton-inference-server/server/blob/master/docs/user_guide/model_configuration.md#dynamic-batcher)
-will be enabled for all model configs generated using automatic search.
+You can also modify the minimum/maximum values that the automatic search space will iterate through:
+
+### [Instance Group Search Space](https://github.com/triton-inference-server/server/blob/master/docs/user_guide/model_configuration.md#instance-groups)
```

Comment:
do we want to be saying instance group here? Or instance count? I think the triton config option is called instance group, but not sure if that is clear to the user in this context. Note that our option name is instance-count. I think we should be consistent and say group or count everywhere.

Reply:
We are split in our documentation about 50/50 between using groups and counts. I think counts makes more sense in the context of MA, but Triton calls these groups.

I'm fine either way, but my preference would be to change group -> count.

Reply:
I'm fine either way, but lets be consistent. You can go ahead and change group to count

Reply:
I actually don't think we can be consistent here. In the YAML it's called `instance-group`, but in our CLI it's called instance count.

We also refer to this as instance group in our tables. 

I now think the best course of action is to change count -> group and consider a story to change the CLI option to use group instead of count. Thoughts?